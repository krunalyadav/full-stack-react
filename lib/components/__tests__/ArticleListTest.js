import React from 'react';
import ArticleList from '../ArticleList';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('ArticleList', () => {
  const testProps = {
    articles: {
      a: { id: 'a' },
      b: { id: 'b' }
    }
  };

  it('renders correctly', () => {
    const wrapper = shallow(<ArticleList {...testProps} />);

    expect(wrapper.find('ArticleContainer').length).toBe(2);
    expect(wrapper).toMatchSnapshot();
  });
});
