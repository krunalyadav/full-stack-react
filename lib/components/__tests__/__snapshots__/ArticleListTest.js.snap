// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleList renders correctly 1`] = `
<div>
  <div
    style={undefined}
  >
    <div
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    />
    <div
      style={
        Object {
          "color": "#888",
          "fontSize": "0.8em",
        }
      }
    >
      Invalid Date
    </div>
    <div
      style={
        Object {
          "paddingBottom": 10,
          "paddingTop": 10,
        }
      }
    >
      <a
        href={undefined}
      >
         
      </a>
    </div>
    <div
      style={
        Object {
          "paddingLeft": 20,
        }
      }
    />
  </div>
  <div
    style={undefined}
  >
    <div
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    />
    <div
      style={
        Object {
          "color": "#888",
          "fontSize": "0.8em",
        }
      }
    >
      Invalid Date
    </div>
    <div
      style={
        Object {
          "paddingBottom": 10,
          "paddingTop": 10,
        }
      }
    >
      <a
        href={undefined}
      >
         
      </a>
    </div>
    <div
      style={
        Object {
          "paddingLeft": 20,
        }
      }
    />
  </div>
</div>
`;
